### ISSUE TITLE
RichTextLabel bbcode refers to the wrong unicode for [ul] bullet points

### ISSUE THREAD

**Issue by craigostrin** [godotengine/godot#75009]
### Godot version

4.0.stable.official

### System information

Ubuntu 22.04.2

### Issue description

When creating an unordered list with RichTextLabel's bbcode, some fonts will display an oversized bullet point, while others appear normal. I'm 99% sure that this is because Godot is referencing the incorrect Unicode for bullet points.

Here are three identical RichTextLabels except for their differing fonts. I used [Franklin Gothic Demi Regular](https://fontsgeek.com/fonts/Franklin-Gothic-Demi-Regular), [Ubuntu Regular](https://fonts.google.com/specimen/Ubuntu), and [Kanit Regular](https://fonts.google.com/specimen/Kanit?category=Sans+Serif,Monospace&noto.script=Latn).

![image](https://user-images.githubusercontent.com/40326682/225772402-ef638eb1-3e41-448d-b927-fd9dbdc2628c.png)

The reason I suspect incorrect unicode is that I dug into the TTF files for these three fonts. They all have normal and similar-sized bullet points at the usual unicode: U+2022. But I believe Godot is actually referencing U+25cf ("black circle"). Franklin Gothic Demi, the font that works as expected, has an entry for U+25cf. The other two fonts do not.

I also found this [other open issue](https://github.com/godotengine/godot/issues/69920) that seems to show that Godot is using U+25cf for bullet points.

### Steps to reproduce

Download a few different fonts, assign them to different RichTextLabels, enable bbcode and type some list items with the `[ul]{ list }[/ul]` format.

### Minimal reproduction project

file:///home/craig/Godot/BulletPoints.zip

**Comment by Calinou** [comment]
cc @bruvzg 

I've noticed this too and would prefer having a smaller bullet point.

**Comment by craigostrin** [comment]
I think it’s literally an error. When I googled it, every site lists 2202 as the Unicode for bullet points.

**Comment by craigostrin** [comment]
By the way, this might just be an issue on Linux. The bullet points look fine on Windows. Haven't tested Mac.

**Comment by bruvzg** [comment]
Indeed, it's using `25CF` (I believe it was selected to display color of the color constants in the docs, and then copy-basted to the rest of docs and RTL). Probably should be customizable.

**Comment by craigostrin** [comment]
On the plus side, I think it should be pretty easy to just copy the character in `2022` to the `25CF` spot using FontForge, as a temporary workaround. But yeah, customizable would be amazing.
